netcode.c

```diff
@@ -6650,11 +6709,30 @@ void test_client_server_multiple_servers()
             netcode_server_free_packet( server, packet );
         }
 
+        // Very basic test for checking if each match only has 2 or less clients connected.
+        skillz_match_t * m;
+        for( m = server->skillz_matches; m != NULL; m = ( skillz_match_t * ) ( m->hh.next ) )
+        {
+            check( m->num_clients_in_match <= server->max_clients_per_match );
+        }
```

Comment:
Should probably use the helper you made above here, right?

I also think it could be called `check_num_clients_in_matches`.

Reply:
Fixed